### ISSUE TITLE
Kubernetes 1.21 support

### ISSUE THREAD

**Issue by pgodwin** [qlik-oss/sense-installer#452]
It looks like most the docs reference Kubernetes 1.16 (Eg the [Getting Started documents](https://github.com/qlik-oss/sense-installer/blob/master/docs/getting_started.md#installing-qliksense-cli)). However 1.16 has been [EOL since 2020-09-02](https://github.com/kubernetes/sig-release/blob/master/releases/patch-releases.md#non-active-branch-history). 

Is QlikSense supported on newer versions of Kubernetes?

**Comment by mickymount** [comment]
@pgodwin did you ever found a work around this show stopper?

**Comment by pgodwin** [comment]
Yes, I reached out to our partner, who in turn asked Qlik and it looks like Kubernetes isn't supported.

**Comment by mickymount** [comment]
I see, thus there is no way qliksense could be containerized. I've spend weeks on this with no progress
